tools/vargo/src/main.rs

```diff
@@ -140,6 +140,13 @@ const Z3_FILE_NAME: &str = if cfg!(target_os = "windows") {
 };
 
 fn run() -> Result<(), String> {
+    let version_info = match util::commit_info() {
+        Ok(info) => info,
+        Err(_) => format!("could not get commit info"),
+    };
+
+    std::env::set_var("VERUS_BUILD_VERSION", version_info);
```

Comment:
The other side of this already marks things as `unknown` if the environment variable is not set, so there is no reason to set the env var in the situation where it couldn't get the info.

Instead, you could do something like this:
```suggestion
    if let Ok(version_info) = util::commit_info() {
      std::env::set_var("VERUS_BUILD_VERSION", version_info);
    }
```

Also, the function is getting you version information as a whole, not just commit info, so probably good to call it `version_info` instead of `commit_info`?